papers/static/style/style.css

```diff
@@ -527,6 +527,11 @@ h1#paperTitle {
     background-image: url('../question-circle.png');
 }
 
+.policyPic-ask
+{
+    background-image: url('../blue-circle.png');

```

Comment:
In case of approval we need a bigger blue circle. I tried to reuse the other svgs to derive a blue circle, but wasn't succesful.

Reply:
"blue circle" is not a very nice name for this file, it is intended to be a dot for lists (`<ul>` type lists, small dot and not centered).

I think you can use https://pub.phyks.me/tmp/blue-circle.svg and https://pub.phyks.me/tmp/blue-circle.png for the circle here.

Reply:
Thanks! I renamed the blue circle to a dot and added your new circles.

Reply:
Nice! Note that if you rename the previous file to a dot, you should also update the code using it currently, as it is expected the dot file and not the new circle one :/

Reply:
I commented too fast, before seeing you added a commit. Everything sould be fine, sorry for the noise :)